packages/protocol/test/stability/stabletoken.ts

```diff
@@ -328,12 +328,30 @@ contract('StableToken', (accounts: string[]) => {
       const [numerator, denominator] = await stableToken.fractionMulExp.call(1, 2, 3, 4, 5, 6)
       assert.equal(numerator.toNumber(), 118652)
       assert.equal(denominator.toNumber(), 1000000)
+
+      const value = await stableToken.mulExp.call(toFixed(1 / 2), toFixed(3 / 4), 5)
```

Comment:
you end up using `floats` here.... maybe this can introduce some flakyness?

Reply:
There's lots of other examples in the tests of passing the return of `toFixed()` (which returns `BigNumber`) to smart contract calls.  Pretty sure this works and `toFixed(x).toFixed()` is not necessary.

Reply:
Seems `toFixed.toFixed()` is needed in the migrations, though.